src/utils.h

```diff
@@ -157,23 +157,23 @@ namespace pyne {
     ~FileNotFound () throw () {};
 
     /// constructor with the filename \a fname.
-    FileNotFound(std::string fname)
+    FileNotFound(std::string fname) : FNF_message("File not found: ")
```

Comment:
Tiny thing, but way as well initialize `filename` and `FNF_message` in the same way.

Reply:
Yes - now do it in the flow (not with initializer) in order to capture to fully set the message using the filename